src/SFA.DAS.ApplyService.Data/AssessorRepository.cs

```diff
@@ -151,6 +151,44 @@ public async Task<int> GetInProgressAssessorApplicationsCount(string userId)
             }
         }
 
+        public async Task<List<RoatpModerationApplicationSummary>> GetApplicationsInModeration()
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                return (await connection
+                    .QueryAsync<RoatpModerationApplicationSummary>(
+                        $@"SELECT 
+                            {ApplicationSummaryFields}
+                            , ModerationStatus AS Status
+	                        FROM Apply apply
+	                        INNER JOIN Organisations org ON org.Id = apply.OrganisationId
+	                        WHERE (Assessor1ReviewStatus = @approvedReviewStatus OR Assessor2ReviewStatus = @approvedReviewStatus) AND ModerationStatus <> @completedModerationStatus
+                            ORDER BY JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ApplicationSubmittedOn')",
+                        new
+                        {
+                            approvedReviewStatus = AssessorReviewStatus.Approved,
+                            completedModerationStatus = ModerationStatus.Complete
+                        })).ToList();
+            }
+        }
+
+        public async Task<int> GetApplicationsInModerationCount()
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                return (await connection
+                    .ExecuteScalarAsync<int>(
+                        $@"SELECT COUNT(1)
+	                      FROM Apply apply
+	                      WHERE (Assessor1ReviewStatus = @approvedReviewStatus OR Assessor2ReviewStatus = @approvedReviewStatus) AND ModerationStatus <> @completedModerationStatus",
+                        new
+                        {
+                            approvedReviewStatus = AssessorReviewStatus.Approved,
+                            completedModerationStatus = ModerationStatus.Complete
+                        }));
+            }
+        }
```

Comment:
I think you need to add 'AND apply.DeletedAt IS NULL' to both of these

Reply:
And possibly an ApplicationStatus check to signify moving from Assessor to Moderation